torch_geometric/nn/pool/approx_knn.py

```diff
@@ -0,0 +1,55 @@
+# This file has code to create a KNN graph with approximate
+# nearest neighbors. We can call it approx-knn.
+
+import torch
+from pynndescent import NNDescent
```

Comment:
This likely needs to be made an optional dependency. You can achieve this by moving the import into the respective function.

Reply:
Done